### PR TITLE
LP-2291 - Fix linting issues in student_certificates application

### PR DESCRIPTION
**Pull request by MunamMubashir-arbisoft**
[LP-2291](https://philanthropyu.atlassian.net/browse/LP-2291)

- Fix pylint issues for the _student_certificates_ application
- Run isort for all the files

6 out 15 tests were failing even prior to the linting changes, kindly ignore those.

### CODE REVIEW

**Review by emraan004 — APPROVED** [review]
Looks good 👍 
There are some code changes/refactoring, please communicate with QA team to test those part in particular.

File: openedx/features/student_certificates/__init__.py [hunk 1]
```diff
@@ -1 +1,4 @@
+"""
+init file for student_certificates ap
```
Comment (harris-irfan) [inline openedx/features/student_certificates/__init__.py:2]: app

File: openedx/features/student_certificates/handlers.py [hunk 2]
```diff
@@ -1,40 +1,56 @@
...
-def generate_certificate_verification_key(instance, created, **kwargs):
+def generate_certificate_verification_key(instance, created, **_kwargs):
     if created and not hasattr(instance, 'certificate_verification_key'):
         CertificateVerificationKey.objects.create_object(instance)
 
 
 @receiver(USER_CERTIFICATE_DOWNLOADABLE)
 def send_email_user_certificate_downloadable(sender, first_name, display_name, certificate_reverse_url, user_email,
-                                             *args, **kwargs):
+                                             *_args, **_kwargs):  # pylint: disable=unused-argument
```
Comment (harris-irfan) [inline openedx/features/student_certificates/handlers.py:32]: Let's please follow the edx [recommendations](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines) regarding breaking long lines.

File: openedx/features/student_certificates/tests/test_views.py [hunk 3]
```diff
@@ -49,7 +50,9 @@ def setUpClass(cls):
         from openedx.features.course_card.tests.helpers import initialize_test_user
         cls.user = initialize_test_user(password=cls.password)
         cls.user.save()
-        cls.course_settings = CustomSettings(id=cls.course.id, course_short_id=1, course_open_date=get_current_utc_date())
+        cls.course_settings = CustomSettings(id=cls.course.id,
+                                             course_short_id=1,
+                                             course_open_date=get_current_utc_date())
```
Comment (harris-irfan) [inline openedx/features/student_certificates/tests/test_views.py:55]: Let's please follow the edx [recommendations](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines) regarding breaking long lines.

**Review by harris-irfan — CHANGES_REQUESTED** [review]
Please have a look at the failing isort check.

File: openedx/features/student_certificates/handlers.py [hunk 4]
```diff
@@ -1,40 +1,56 @@
...
+    Arguments:
+        sender: Sender that triggered or sent the signal
+        first_name (str): First name of the user
+        display_name (str): Display name of the user
+        certificate_reverse_url (str): Url of the downloadable ceritificate
+        user_email (str): Email of the recipient user
+
+    Returns:
+        None
+
```
Comment (annas-usman) [inline openedx/features/student_certificates/handlers.py:46]: Lets remove this extra line

File: openedx/features/student_certificates/helpers.py [hunk 5]
```diff
@@ -145,6 +156,17 @@ def get_credential_certificates(user):
...
+    """
+    Get course certificate urls for sharing on social sites
+
+    Arguments:
+         course (Course): Course which the urls are associated with
+         certificate (GeneratedCertificate): Certificate for which to generate social urls
+
+    Returns:
+        dict: Dictionary containing the urls (value) for different social sites (key)
+
```
Comment (annas-usman) [inline openedx/features/student_certificates/helpers.py:168]: Lets remove this extra line as well

File: openedx/features/student_certificates/models.py [hunk 6]
```diff
@@ -27,8 +33,18 @@ class CertificateVerificationKey(models.Model):
...
     def generate_hash(certificate):
+        """
+        Generate a hash for the specified certificate
+
+        Arguments:
+            certificate (GeneratedCertificate): Certificate to generate a hash for
+
+        Returns:
+            str: Hash key to verify
+
```
Comment (annas-usman) [inline openedx/features/student_certificates/models.py:44]: Lets remove this extra line as well

**Review by annas-usman — CHANGES_REQUESTED** [review]

**Review by annas-usman — APPROVED** [review]
LGTM

**Comment by MunamMubashir-arbisoft** [comment]
The failing isort command is due to skipping one file for some unknown reason.
However, isort has been run of all the files locally and no error is being reported over there.
@harris-irfan

**Review by harris-irfan — APPROVED** [review]

**Pull request merged**